app/scripts/require_config.js

```diff
@@ -25,6 +25,7 @@ require.config({
     backbone: '../bower_components/backbone/backbone',
     canvasToBlob: '../bower_components/blueimp-canvas-to-blob/js/canvas-to-blob',
     chai: '../bower_components/chai/chai',
+    'fxa-checkbox': '../bower_components/fxa-checkbox/checkbox',
```

Comment:
-1 because we already have so many non camelCase:

"fxa-checkbox": "mozilla/fxa-checkbox#fd04a1b2621bc68feb8f3436cfc8b801c88fd53f",
    "fxa-content-server-l10n": "https://github.com/mozilla/fxa-content-server-l10n.git",
    "fxa-js-client": "https://github.com/mozilla/fxa-js-client.git#0.1.30",
    "fxa-password-strength-checker":